### PR TITLE
Prep for `v2` release

### PR DESCRIPTION
**Pull request by ianrohde**
Summary of changes
---

- Add keys to ~~`Space`~~, `FontSize`~~, and `Shadow`~~
- Update readme with new formats
- Remove `main.workflow` ([Issue](https://github.com/iFixit/core-primitives/issues/15))
- Add trailing blank lines to built stylesheets
- Add `color.blue.ifixit` https://github.com/iFixit/core-primitives/pull/23/commits/d9e53d25ab5714731e73dd053abbb9e498a5dbaf
- Remove unnecessary gray color families https://github.com/iFixit/core-primitives/pull/23/commits/a851cee4c8f479533ade298be38b44bbdb31c089
- Update kebab-case package:

| Old Generated           | New Generated          |
| ----------------------- | ---------------------- |
| `...`   | `...` |
| `--font-size-2-xl: 20px;` | `--font-size-2xl: 20px;` |
| `--font-size-3-xl: 24px;` | `--font-size-3xl: 24px;` |
| `--font-size-4-xl: 30px;` | `--font-size-4xl: 30px;` |
| `--font-size-5-xl: 36px;` | `--font-size-5xl: 36px;` |
| `--font-size-6-xl: 48px;` | `--font-size-6xl: 48px;` |
| `--font-size-7-xl: 56px;` | `--font-size-7xl: 56px;` |
| `--font-size-8-xl: 64px;` | `--font-size-8xl: 64px;` |
| `--font-size-9-xl: 72px;` | `--font-size-9xl: 72px;` |

Linked to: https://github.com/iFixit/ifixit/issues/39221

Merge checklist
---

- [x] Bump the package version by running `npm version [patch | minor | major]` from the command line (if applicable).

> **Note:** In the context of Core Primitives, significant changes to the library or workflow, or removing primitives would be considered a major update, adding or updating primitives would be considered a minor update, and fixing primitives would be considered a patch. Non-code changes (e.g. documentation) do not require a version bump.

qa_req 0

### CODE REVIEW

**Review by bhuminson — APPROVED** [review]

**Comment by bhuminson** [comment]
CR :dagger:

**Comment by BaseInfinity** [comment]
CR 👙

**Comment by deltuh-vee** [comment]
Is there anything to QA for this?

**Comment by ianrohde** [comment]
> Is there anything to QA for this?

No, sorry about that 😬

**Review by bhuminson — APPROVED** [review]

**Comment by bhuminson** [comment]
CR :dagger:

**Comment by BaseInfinity** [comment]
CR 👙

**Comment by ianrohde** [comment]
Prepping for a v2.1.0 release. The deprecated v2.0.0 is causing issues that need to be resolved.

**Comment by ianrohde** [comment]
deploy_block 🛑 need to consider how to best deal with this

**Comment by bhuminson** [comment]
CR :dagger: with a question -- I remember a comment on some pull a while back saying that we're trying to stay away from `em`, does that not apply to core primitives?

**Comment by ianrohde** [comment]
> CR 🗡️ with a question -- I remember a comment on some pull a while back saying that we're trying to stay away from `em`, does that not apply to core primitives?

The `em` for the pill radii is a multiplier just in case we ever need to use it on anything over 999px high. It's certainly a stretch that we'll ever need it, but I always try to account for edge cases when I see the opportunity.

**Comment by ianrohde** [comment]
dev_block 🛑 

Working on an RC version for testing

**Comment by ianrohde** [comment]
un_deploy_block 🟢 
un_dev_block 🟢 

Ready for 2.2.0 release, which will be needed for opening the color conversion pull

**Review by bhuminson — APPROVED** [review]
CR :dagger:

File: index.d.ts [hunk 1]
```diff
@@ -11,102 +11,60 @@ interface Primitives {
...
+  "300": string;
+  "400": string;
+  "500": string;
+  "600": string;
+  "700": string;
+  "800": string;
+  "900": string;
+}
+
+interface ColorValueBlue {
```
[the remark this answers is not in the document]
Comment (ianrohde) [reply index.d.ts:27]: TIL - thanks!

**Comment by sctice-ifixit** [comment]
CR :flags: I wouldn't say I'm super confident reviewing this; I have a hard time anticipating what could go wrong. Overall, things look sane. Can you give an idea of the scope of these changes? If I understand correctly, this is sort of prep work for a big change to actually use the new version and update a bunch of references to constants?

deploy_block :hand: on considering some changes.

More generally, the commit messages here are more "what" than "why", and it's addressing the "why" that will better serve review and future efforts to recover history. I think points 3–5 in [this summary of what makes for an excellent commit message](https://thoughtbot.com/blog/5-useful-tips-for-a-better-commit-message) are very helpful.

**Comment by sctice-ifixit** [comment]
CR :flags:

**Comment by ianrohde** [comment]
> If I understand correctly, this is sort of prep work for a big change to actually use the new version and update a bunch of references to constants?

Yes, exactly. 👍

**Comment by ianrohde** [comment]
> deploy_block ✋ on considering some changes.

Aside from https://github.com/iFixit/core-primitives/pull/23/commits/c7785668dcc84376e1a3e10a29b2e6fafdf3bfbc, were there any other changes?

**Comment by sctice-ifixit** [comment]
> were there any other changes?

un_deploy_block :flags: Sorry, nope!

**Comment by ianrohde** [comment]
Published: https://www.npmjs.com/package/@core-ds/primitives/v/2.2.0

**Pull request merged**